.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
         aws_auth:
           aws_access_key_id: $AWS_ACCESS_KEY_ID
           aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
+    resource_class: small
```

Comment:
![](https://media2.giphy.com/media/jrutBd1N7ZhsINAPzs/giphy.gif)

Reply:
@JessicaMarine , I'm tagging you to see that we're changing our circleci resource utilization, in case changes to the payment plan can be made based on these changes.